gcc-4.8/post-link.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/sh
+
+if [ "$(uname)" != "Darwin" ]; then
+
+    build_os_md5=( `md5sum ${PREFIX}/share/conda-gcc-build-machine-os-details` )
+    target_os_md5=( `cat /etc/*-release | md5sum` )
+
+    # No need to make any portability fixes if 
+    # we're deploying to the same OS we built with.
+    if [[ "${build_os_md5[0]}" == "${target_os_md5}" ]]; then
+        echo "gcc install OS matches gcc build OS: Skipping post-link portability fixes."
+    else
+
+        # In this script, we attempt to fix 3 Linux distro portability issues:
+    
+        #
+        # Linux Portability Issue #1: "fixed includes"
+        #
+    
+        # Remove the headers that gcc "fixed" as part of the gcc build process.
+        # They kill the gcc binary's portability to other systems,
+        #   and shouldn't be necessary on ANSI-compliant systems anyway.
+        # More discussion can be found here:
+        # https://groups.google.com/a/continuum.io/d/msg/conda/HwUazgD-hJ0/aofO0vD-MhcJ
+        while read x ; do
+          grep -q 'It has been auto-edited by fixincludes from' "${x}" \
+                   && rm -f "${x}"
+        done < <(find ${PREFIX}/lib/gcc/*/*/include*/ -name '*.h')
```

Comment:
The latest version of shellcheck gives this error message:

```
In gcc-4.8/post-link.sh line 28:
        done < <(find ${PREFIX}/lib/gcc/*/*/include*/ -name '*.h')
               ^-- SC2040: In sh, process substitution is not supported, even when sh is actually bash.
                      ^-- SC2086: Double quote to prevent globbing and word splitting.
```